Civi/Api4/LocBlock.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4;
+
+/**
+ * ContributionPage entity.
```

Comment:
@eileenmcnaughton can you do a follow up and fix this up unless it has already done so otherwise it will look odd in the APIv4 Explorer

Reply:
opps - yes!